motr/m0crate/crate_index.c

```diff
@@ -237,22 +281,12 @@ static void cr_time_measure_begin(struct cr_time_measure_ctx *t)
 
 static double cr_time_measure_elapsed_now(struct cr_time_measure_ctx *t)
 {
-	m0_time_t d = m0_time_sub(m0_time_now(), t->ts);
+	t->ts_next = m0_time_now();
+	m0_time_t d = m0_time_sub(t->ts_next, t->ts);
 	return (m0_time_seconds(d)) +
 		( m0_time_nanoseconds(d) / (double) M0_TIME_ONE_SECOND );
 }
 
-static void cr_time_measure_end(struct cr_time_measure_ctx *t)
```

Comment:
suggestion is, cr_time_measure_end() is useful to have and not to change cr_time_measure_elapsed_now(). We can have cr_time_measure_elapsed() which can be called after cr_time_measure_end() to get the time after timer is stopped and cr_time_measure_elapsed_now() can be used to measure intermediate time without stopping the timer.

Reply:
Thanks, as per suggestion keeping original  changes. cr_time_capture_result() will prepare final results.